### PR TITLE
Update the conditionals buttons and tooltips on covers to reflect their actual effects

### PR DESCRIPTION
**Pull request by C0bra5**
Closes: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14258

- Added button overlays for buttons that affect behaviours controlled by machine state.
  - It's a lever, in reference to the machine controller cover.
  - There is an upside-down version for when the behaviour is inverted.
- Added new language lines for "Use Machine Processing State", "Use Inverted Machine Processing State", and "Always Off"
- Mentions of Redstone in the tooltips of the conditional buttons have been replaced with machine processing state on the following covers: Pump, Fluid Regulator, Steam Valve, Steam Regulator and Conveyor belt.
- Updated the buttons overlays on the previously mentioned covers to the new machine processing state button overlays.
- Updated the conveyor belt buttons to better reflect their actual effects in the game
  - This also highlights the issue that the first 2 buttons do the same thing while the third completely disabled the belt.

image
image
image

Here is a chart of the current effects of the machine state on the affected covers as observed in GT:NH v2.3.6

image

### CODE REVIEW

**Comment by Lyfts** [comment]
There's a typo in machine in the inverted portion and a typo in processing in the other.

Also why not fix the conveyor to bring it in line with the other covers rather than leaving it with 2 identical buttons?

**Comment by C0bra5** [comment]
> There's a typo in machine in the inverted portion and a typo in processing in the other.
I'll fix that. Though Idk how yall handle spotless either, 

> Also why not fix the conveyor to bring it in line with the other covers rather than leaving it with 2 identical buttons?
I'm sure how yall wanted to handle the fix for this. I'd be willing to bring it in line, but I'm worried doing so may break some existing setups.

**Comment by C0bra5** [comment]
Upon further testing, I realized that the inconsistency with the conveyor belts mentioned previously was in fact a result of my testing methodology. Further testing with a different methodology has proven the conveyors to work exactly like the rest. So no changes should occur to functionality.

**Comment by C0bra5** [comment]
https://github.com/GTNewHorizons/GT5-Unofficial/pull/2244#issuecomment-1684894176
The typos should be fixed; I also made the blocking section more accurate to its actual effect and updated the icons to be more easily recognizable.

https://github.com/GTNewHorizons/GT5-Unofficial/assets/17766417/95efe5b3-f8c9-46e2-81e9-8db7891856ef

**Review by Lyfts — COMMENTED** [review]
Overall i think this change makes the function of the buttons a lot more clear than before. Great work.

File: src/main/java/gregtech/api/util/GT_LanguageManager.java [hunk 1]
```diff
@@ -506,6 +510,7 @@ public static void writePlaceholderStrings() {
         addStringLocalization("Interaction_DESCRIPTION_Index_756", "Connectable: ");
         addStringLocalization("Interaction_DESCRIPTION_Index_ALL", "All");
         addStringLocalization("Interaction_DESCRIPTION_Index_ANY", "Any");
+        addStringLocalization("Interaction_DESCRIPTION_Index_ANY", "Any");
```
Comment (miozune) [inline src/main/java/gregtech/api/util/GT_LanguageManager.java:513]: duplicated addition?

Comment (C0bra5) [reply src/main/java/gregtech/api/util/GT_LanguageManager.java:513]: likely

**Comment by C0bra5** [comment]
There we go all the typos, and that weird extra line have all been removed

**Review by miozune — APPROVED** [review]

**Pull request merged**